commit 6cace8b8ecedd11a54b02e1dd08f9a2880bdb1b1
Author: Isaac Dunham <ibid.ag@gmail.com>
Date:   Tue Nov 18 21:06:54 2014 -0800

    Don't use FLTK internal symbols; fix build with FLTK 1.3.3
    Patch based on http://hg.dillo.org/dillo/raw-rev/eb902ac9fc66

diff --git a/dw/fltkviewbase.cc b/dw/fltkviewbase.cc
index 30a19d5..9ec5a71 100644
--- a/dw/fltkviewbase.cc
+++ b/dw/fltkviewbase.cc
@@ -27,8 +27,6 @@
 #include <stdio.h>
 #include "../lout/msg.h"
 
-extern Fl_Widget* fl_oldfocus;
-
 using namespace lout::object;
 using namespace lout::container::typed;
 
@@ -364,7 +362,10 @@ int FltkViewBase::handle (int event)
       }
       return 1;
    case FL_UNFOCUS:
-      focused_child = fl_oldfocus;
+      // was: focused_child = fl_oldfocus;
+      for (Fl_Widget *p = this; p; p = p->parent())
+        focused_child = p;
+
       return 0;
    case FL_KEYBOARD:
       if (Fl::event_key() == FL_Tab)
